Community-Supported/publish-hyper/publish-hyper-file.py

```diff
@@ -0,0 +1,109 @@
+from pathlib import Path
+import tableauserverclient as TSC
+from tableauhyperapi import HyperProcess, Telemetry, \
+    Connection, CreateMode, \
+    NOT_NULLABLE, NULLABLE, SqlType, TableDefinition, \
+    Inserter, \
+    escape_name, escape_string_literal, \
+    TableName, \
+    HyperException
+
+
+# Configure for TSC to publish
+# Note: Do not store creds/tokens in plaintext, please use env vars :)
+hyper_name = 'customer.hyper'
+server_address = 'https://10ax.online.tableau.com/'
+site_name = 'mysitename'
+project_name = 'projectname'
```

Comment:
Minor: you call the first one my.. the other one just project... maybe call mysite and myproject?